src/utils/initializeCaptainsConfig.js

```diff
@@ -32,25 +28,29 @@ const initialize = config => {
     slack_message_heading,
   } = conf;
 
-  // Github
-  config.set('github:domain', enterprise_host);
-  config.set('github:host', github_host);
-  config.set('github:owner', github_owner);
-  config.set('github:repo', github_repo);
-  config.set('github:tagId', github_tag_id);
-  config.set('github:token', github_token);
-
-  // Team Configuration
-  config.set('teams', teams);
-
-  // Slack
-  config.set('slack:channel', slack_channel);
-  config.set('slack:token', slack_token);
-  config.set('slack:channelUrl', slack_url);
-  config.set('slack:messageHeading', slack_message_heading);
-
-  // Jira
-  config.set('jira:teamDomain', jira_team_domain);
+  const captainsConfig = {
+    // Github
+    enterprise_host: process.env.PLUGIN_ENTERPRISE_HOST || enterprise_host,
```

Comment:
Would it be a little cleaner to use destructuring defaults off on `conf` above here for the `process.env` overrides?

Reply:
I actually went this route initially, but the issue of "combined" usage is not an edge case. I found that if I wanted to spread `process.env` over `conf` it resulted in `undefined` for most values if they were mostly defined in `conf`. This is not ideal, so I went to an `OR` implementation for simplicity. This also assures that `process.env` vars are respected as first priority, which is listed in the documentation now.